### PR TITLE
Gitlab Readme and default password update

### PR DESCRIPTION
**Pull request by jfilipcz**
* Updated README to cover Gitlab integration with LDAP
* Updated root password for Gitlab;)

### CODE REVIEW

File: tooling/charts/tl500-base/values.yaml [hunk 1]
```diff
@@ -78,7 +78,7 @@ logging:
 
 gitlab:
   namespace: tl500-gitlab
-  root_password: Password123
+  root_password: 7aydhn160bOrrsGEbnd172rE
```
Comment (pabrahamsson) [inline tooling/charts/tl500-base/values.yaml:81]: Can you update this in [tooling/charts/tl500-base/templates/gitlab/deployments.yaml#L64](https://github.com/rht-labs/enablement-framework/blob/main/tooling/charts/tl500-base/templates/gitlab/deployments.yaml#L64) as well?

**Review by pabrahamsson — CHANGES_REQUESTED** [review]
This is great, thanks @jfilipcz 
I've requested two minor changes.

**Review by pabrahamsson — APPROVED** [review]
LGTM

**Comment by eformat** [comment]
@pabrahamsson @jfilipcz - there is NO valid reason to change the default password ?
security through obscurity never works.
Pass this parameter to the chart when installing if you dont want the defaults !!

**Pull request merged**